scss-test.scss

```diff
@@ -0,0 +1,16 @@
+.unsorted {
+  color: #FFFFFF;
+  background: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.